text/0061-error-format-and-definitions.md

```diff
@@ -468,6 +468,19 @@ HTTP Code: `400 Bad Request`
 - The `:attribute` is inferred when the message is generated.
 - The `:filterableAttributes` is inferred when the message is generated. It contains the list of filterable attributes separated by a comma. `filterableAttribute1, filterableAttribute2, ...`
 
+#### Variant: Filtering on a non filterable attribute when `filterableAttributes` is empty
+
+```json
+{
+    "message": "Attribute `:attribute` is not filterable. This index does not have configured filterable attributes.",
+    "code": "invalid_filter",
+    "type": "invalid_request",
+    "link": "https://docs.meilisearch.com/errors#invalid_filter"
+}
+```
+
+- The `:attribute` is inferred when the message is generated.
```

Comment:
Minor suggestion: English generally requires a lot less definite articles than other languages. In this case, you could safely drop the first `the`:

```suggestion
- `:attribute` is inferred when the message is generated.
```

Reply:
Thanks @gui, I'll make a patch that removes all that before releasing v0.27!